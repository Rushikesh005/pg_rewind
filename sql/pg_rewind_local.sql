# This file has the .sql extension, but it is actually launched as a shell
# script. This contortion is necessary because pg_regress normally uses
# psql to run the inptu scripts, and requires them to have the .sql
# extension, but we use a custom launcher script that runs the scripts using
# a shell instead.
sql/pg_rewind.sh --local
