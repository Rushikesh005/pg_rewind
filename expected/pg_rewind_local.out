\! sh sql/pg_rewind.sh --local
              d              
-----------------------------
 in master
 in master, before promotion
 in standby, after promotion
(3 rows)

